extensions/physics-nodes/CCPhysicsDebugNodeBox2D.h

```diff
@@ -35,6 +35,66 @@ struct b2AABB;
 
 extern cocos2d::DrawNode* drawBox2D;
 
+/// Test settings. Some can be controlled in the GUI.
+struct Settings
+{
+    Settings()
+    {
+        viewCenter.Set(0.0f, 0.0f);
+        hz = 60.0f;
+        velocityIterations = 8;
+        positionIterations = 3;
+        drawShapes = 1;
+        drawJoints = 1;
+        drawAABBs = 0;
+        drawContactPoints = 0;
+        drawContactNormals = 0;
+        drawContactImpulse = 0;
+        drawFrictionImpulse = 0;
+        drawCOMs = 0;
+        drawStats = 0;
+        drawProfile = 0;
+        enableWarmStarting = 1;
+        enableContinuous = 1;
+        enableSubStepping = 0;
+        enableSleep = 1;
+        pause = 0;
+        singleStep = 0;
+    }
+
+    b2Vec2 viewCenter;
+    float hz;
+    int32 velocityIterations;
+    int32 positionIterations;
+    int32 drawShapes;
+    int32 drawJoints;
+    int32 drawAABBs;
+    int32 drawContactPoints;
+    int32 drawContactNormals;
+    int32 drawContactImpulse;
+    int32 drawFrictionImpulse;
+    int32 drawCOMs;
+    int32 drawStats;
+    int32 drawProfile;
+    int32 enableWarmStarting;
+    int32 enableContinuous;
+    int32 enableSubStepping;
+    int32 enableSleep;
+    int32 pause;
+    int32 singleStep;
+};
+
+struct ContactPoint
```

Comment:
put to namespace cocos2d::extensions maybe better

Reply:
I think Settings will be removed in the next clean up.
For the moment I improve/clean up the Box2D - TestBed.

Can you accept the pull request at the moment?

Reply:
Accepted